packages/weather.rb

```diff
@@ -0,0 +1,37 @@
+require 'package'
+
+class Weather < Package
+  description 'This command-line utility is intended to provide quick access to current weather conditions and forecasts.'
+  homepage 'http://fungi.yuggoth.org/weather/'
+  version '2.3'
+  source_url 'http://fungi.yuggoth.org/weather/src/weather-2.3.tar.xz'
+  source_sha256 '86148d2f1d59867f637f52558cc2a6b3280fac94df55c6e5af0ce37cc190d146'
+
+  depends_on 'python27' unless File.exists? "#{CREW_PREFIX}/bin/python"
+
+  def self.build
+    system "echo 'setpath = $HOME/.weather:#{CREW_PREFIX}/data/weather' >> weatherrc"
+    system "if [ ! -d $HOME/.weather ]; then mkdir $HOME/.weather; fi"
+    system "if [ ! -f $HOME/.weather/weatherrc ]; then cp weatherrc $HOME/.weather; fi"
```

Comment:
modifications of $HOME in build process.

Reply:
What do you think about the differences between build function and install function?

Reply:
So you want me to move this to the install function?  Please explain your issue more clearly.

Reply:
Yes.  That's what I want.  I can ask you to do so.  But if you don't understand why, you will do it again.  Therefore, I would like to understand why you put those stuff into build function instead of install function.  You submitted a lot of PRs, so I believe you understand something like below...

 - build: configure and make (compile)
 - install: make install or file copies
 - postinstall: similar to install, but install is executed only for source build.  so, anything need to be executed for both source and binary, put them here.

Reply:
It's totally arbitrary and I completely understand why.  Please don't sound patronizing.  This is definitely a build command: `system "echo 'setpath = $HOME/.weather:#{CREW_PREFIX}/data/weather' >> weatherrc"`.  I could agree the `$HOME` stuff might be better in the install function.

Reply:
I agree about generating `weatherrc`.  In addition, we can use postinstall function for those `$HOME` stuff now.  It's available now.

Reply:
Done.